packages/jellyfish-transaction/src/script/dftx/dftx_icxorderbook.ts

```diff
@@ -1,5 +1,6 @@
 import { BufferComposer, ComposableBuffer } from '../../buffer/buffer_composer'
 import BigNumber from 'bignumber.js'
+import { SmartBuffer } from 'smart-buffer'
 
 // Disabling no-return-assign makes the code cleaner with the setter and getter */
```

Comment:
You may remove line 5 and 6. 
You cannot see them anymore in other code.

Reply:
https://eslint.org/docs/rules/no-return-assign